dist/s_style

```diff
@@ -46,6 +46,13 @@ else
 		if egrep -r -n '\b[a-z]+[A-Z]' $f | egrep -v ':[     ]+\*|"|UNCHECKED_STRING'; then
 			echo "$f: Styling requires variables that use underscores to separate parts of a name instead of camel casing.";
 		fi
+		egrep -r -n '^[^*/"]*[[:space:]]*return [^(]' $f > $t;
```

Comment:
Nitpick: there should be a line break here to separate it from the logically separate task of picking up camel case errors. The comments around here could also a little clearer now that you've added some more functionality, e.g.:

```
# Ignore styling errors of external libraries
if expr "$f" : 'src/' > /dev/null &&
  ...

  # Camel case style is not allowed.
  if egrep -r -n '\b[a-z]+[A-Z]' $f | egrep -v ':[     ]+\*|"|UNCHECKED_STRING'; then
  ...

  # Return values must be wrapped in parentheses.
  egrep -r -n '^[^*/"]*[[:space:]]*return [^(]' $f > $t;
  ...
```

Reply:
You make a very good point! I forgot to add commenting,  I will adjust this now.